README.md

```diff
@@ -1,25 +1,106 @@
+|------------|----------------------|----------------------------------|----------------------------------------------------|---------------------------------------------------------|
+| -----------| ---------------------| ---------------------------------| ---------------------------------------------------| --------------------------------------------------------|
+| GCP        | [cmd/gcp](./cmd/gcp/)| [VM](./deployment/live/gcp/test/)| [Cloud Run](deployment/live/gcp/static-ct/logs/ci/)| [Cloud Run](deployment/live/gcp/static-ct-staging/logs/)|
+| AWS        | [cmd/aws](./cmd/aws/)| [VM](./deployment/live/aws/test/)| [Fargate](deployment/live/aws/test/)               |                                                         |
+
+## 🙋 FAQ
+
+### TesseraWhat?
+
+TesseraCT is the concatenation of Tessera and CT (Certificate Transparency),
+which also happens to be a [4-dimensional hypercube](https://en.wikipedia.org/wiki/Tesseract).
+
+### What's the difference between Tessera and TesseraCT?
+
+[Tessera](https://github.com/transparency-dev/tessera) is a Go library for
+building [tile-based transparency logs (tlogs)](https://c2sp.org/tlog-tiles) on
+various deployment backends. TesseraCT is a service using the Tessera library
+with CT specific settings to implement Certificate Transparency logs complying
+with [static-ct-api](https://c2sp.org/static-ct-api). TesseraCT supports a
+subset of Tessera's backends. A TesseraCT serving stack is composed of:
+
+- one or multiple instances of a TesseraCT binary using the Tessera library
+- Tessera's backend infrastructure
+- a minor additional storage system for [chain issuers](https://github.com/C2SP/C2SP/blob/main/static-ct-api.md#issuers)
+
+### Why these backends?
+
+After chatting with various CT log operators, we decided to focus on GCP and AWS
+to begin with in an effort address current needs of log operators. We're
+welcoming contributions and requests for additional backend implementations.
+If you're interested, [come and talk to us](#-contact)!
+
+## 🧌 History
+
+TesseraCT is the successor of [Trillian's CTFE](https://github.com/google/certificate-transparency-go/tree/master/trillian/ctfe).
+It was built upon its codebase, and introduces these main changes:
+
+- **API**: TesseraCT implements [static-ct-api](https://c2sp.org/static-ct-api)
+rather than [RFC6962](https://www.rfc-editor.org/rfc/rfc6962).
+- **Backend implementation**: TesseraCT uses [Tessera](https://github.com/transparency-dev/tessera)
+rather than [Trillian](https://github.com/google/trillian). This means that
+TesseraCT integrates entries faster, is cheaper to maintain, requires running a
+single binary rather than 3, and does not need additional services for leader election.
+- **Multi tenancy**: One TesseraCT instance serves a single CT log, as opposed
```

Comment:
```suggestion
- **Single tenancy**: One TesseraCT instance serves a single CT log, as opposed
```

Brings this into line with bullets above structured in a _"TesseraCT is/has <property now> rather than <property before>"_ fashion.